cameo/api/designer.py

```diff
@@ -170,7 +170,7 @@ def __call__(self, product='L-glutamate', hosts=HOSTS, database=None, aerobic=Tr
         notice("Starting searching for compound %s" % product)
         try:
             product = self.__translate_product_to_universal_reactions_model_metabolite(product, database)
-        except KeyError:
+        except Exception:
```

Comment:
maybe first catch the KeyError then the catch-all and re-raise it with with some added info rather than converting everything to KeyError?